bundles/core/org.eclipse.smarthome.core/ESH-INF/config/networkinterface.xml

```diff
@@ -16,6 +16,16 @@
 			<limitToOptions>true</limitToOptions>
 			<advanced>true</advanced>
 		</parameter>
+		<parameter name="useOnlyOneAddress" type="boolean">
```

Comment:
Could you please also add a default value here?

Reply:
Isn't false the default? Why do I need to remove the default attribute value from the ds annotation but explicitly add the default value here?